apps/site/components/Common/AvatarGroup/Avatar/index.tsx

```diff
@@ -11,16 +13,18 @@ export type AvatarProps = {
 
 const Avatar: FC<AvatarProps> = ({ src, alt, fallback }) => (
   <RadixAvatar.Root className={styles.avatarRoot}>
```

Comment:
`RadixAvatar.Root` becomes a `span`. I don't think it matters from a DOM validation part which is outer-most - so I left Root, root. It also worked moving all the styles to the Link when I briefly had that as the root.